README.md

```diff
@@ -38,6 +38,13 @@ Use `yarn test` to run unit tests. `yarn test-int` will run the integration test
 
 We document our decisions in the *doc/adr/* directory. Use [adr-tools](https://github.com/npryce/adr-tools) to automate creating a new decision register file for your decision. More details in the [wiki](https://github.com/architecture-topography/topo/wiki/decision-register).
 
+## yarn tasks
```

Comment:
I'll use this section as a way to document the yarn tasks available as I go along